src/styles/colors.js

```diff
@@ -10,5 +10,6 @@ export default {
     green: '#03d47c',
     greenHover: '#03c775',
     red: '#fc3826',
+    yellow: '#FED607',
```

Comment:
Minor comment but can we use all lowercase here for consistency? `#fed607`

Reply:
Updated